stable/jenkins/templates/jcasc_config.yaml

```
@@ -0,0 +1,18 @@
+{{- $root := . }}
+{{- if and (.Values.Master.JCasC.enabled) (.Values.Master.Sidecar.configAutoReload.enabled) }}
+{{- range $key, $val := .Values.Master.JCasC.ConfigScripts }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jenkins-config-{{ template "jenkins.fullname" $root }}-{{ $key }}
+  labels:
+    {{ $.Values.Master.Sidecar.configAutoReload.label | default "jenkins_config" }}: "true"
+    release: {{ $root.Release.Name }}
+    chart: "{{ $root.Chart.Name }}-{{ $root.Chart.Version }}"
+    component: "{{ $root.Release.Name }}-{{ $.Values.Master.Name }}"
+data:
+  {{ $key }}.yaml: |-
+{{ $val | indent 4 }}
```

```suggestion
{{ tpl $val $| indent 4 }}
```
This would allow templating for JCasC config snippets, such as for the welcome message to include the name of the instance:
`
    ConfigScripts:
      welcomemessage: |
        jenkins:
          systemMessage: "{{ template "jenkins.fullname" . }} configured automatically by Helm Chart & Jenkins Configuration as Code Plugin\n\n"
`